app-backup/zbackup/zbackup-1.4.4-r2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A versatile deduplicating backup tool"
+HOMEPAGE="https://github.com/zbackup/zbackup"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+-with-openssl-exception"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
```

Comment:
please swap those two lines to be in line with ebuild writing standards/skel.ebuild.

Reply:
Done